dmutils/email.py

```diff
@@ -57,12 +57,12 @@ def generate_token(data, secret_key, salt):
     return ts.dumps(data, salt=salt)
 
 
-def decode_token(token, secret_key, salt):
+def decode_token(token, secret_key, salt, max_age=86400):
```

Comment:
I would have said max-age as a number of seconds is pretty common on the web (HTTP cache headers, cookie lifetimes). I'd rather it stays as seconds as it's then more instantly comparable to these other numbers.